Quality/MK8_1080p/rules.txt

```diff
@@ -18,6 +18,22 @@ formatsExcluded = 0x41A # exclude obvious textures
 overwriteWidth = 960
 overwriteHeight = 540
 
+# Multiplayer-Mode Resolution (2p)
+[TextureRedefine]
+width = 640
+height = 720
+formatsExcluded = 0x41A # exclude obvious textures
+overwriteWidth = 960
+overwriteHeight = 1080
+
+# Multiplayer-Mode Resolution (4p)
+[TextureRedefine]
+width = 640
+height = 360
+formatsExcluded = 0x41A # exclude obvious textures
+overwriteWidth = 960
+overwriteHeight = 1080
```

Comment:
Alright I see what happened with the "conflict", I wasn't using the current half-res alpha rule in my own Cemu graphic pack folder when I was trying to fix MK8 4 player multiplayer resolution. Apparently my "4P multiplayer" rule isn't needed as the existing half-res alpha rule also fixes the same issue. 
Tbh I wanted to keep the half-width normal height format(?) used for 2 player splitscreen/multiplayer in 4 player splitscreen/multiplayer but I'll just make a higher res graphic pack instead and avoid any potential half-res issues all together.